src/main/scala/edu/ie3/osmogrid/io/input/InputDataProvider.scala

```diff
@@ -70,12 +74,13 @@ object InputDataProvider extends ActorStopSupport[ProviderData] {
   ): Behavior[InputDataEvent] = {
     Behaviors.withStash[InputDataEvent](100) { buffer =>
       Behaviors.setup[InputDataEvent] { ctx =>
+        implicit val ec: ExecutionContextExecutor = ctx.system.executionContext
```

Comment:
This val does not have to be implicit. Generally, this does no harm, it's just that implicits are a little icky because they're hard to trace, so I try to avoid them